tests/cat/aliases.yml

```diff
@@ -19,5 +19,6 @@ teardown:
 ---
 "cat.aliases":
   - do:
-      cat.aliases: {}
-  - match: { $body.0.alias: 'my_alias' }
+      cat.aliases:
+        format: 'json'
+  - match: { 0.alias: 'my_alias' }
```

Comment:
All of the tests pass for JS except this one, because `my_alias` is not first in the list. There are three hidden indices that show up before it. So maybe we need to be explicit about `expand_wildcards` as well.